bindings/python/setup.py

```diff
@@ -41,4 +53,5 @@ def get_py_version_cfgs():
     ],
     include_package_data=True,
     zip_safe=False,
+    install_requires=load_requirements("requirements.txt")
```

Comment:
You know if this file would get published? Because the dev.txt doesn't

Also maybe we should just use https://github.com/PyO3/maturin#python-metadata instead?

Reply:
Ideally it should build and paste in the array like format. But since right now there's only 2, can do it directly.
Updating.